docker-compose.yml

```diff
@@ -35,5 +35,21 @@ services:
     image: "${DOCKER_IMAGE}-test-standalone-chrome:${TAG:-latest}"
     environment:
       - CI=true
+      - SITE=build
+    volumes:
+      - $HOME/$TRAVIS_BUILD_ID/build:/opt/module/build
+    depends_on:
+      - standalone-chrome
+    command: 'npm run wdio-default'
+
+  test-ci-lowlight:
+    build: ./
+    image: "${DOCKER_IMAGE}-test-standalone-chrome:${TAG:-latest}"
+    environment:
+      - CI=true
+      - SITE=build
+    volumes:
+      - $HOME/$TRAVIS_BUILD_ID/build:/opt/module/build
     depends_on:
-      - standalone-chrome
\ No newline at end of file
+      - standalone-chrome
+    command: 'npm run wdio-lowlight'
```

Comment:
@mjhenkes @emilyrohrbough ,
The reason I added a separate service for clinical-lowlight-theme is because, the default command is being overriden in "**test-ci**" service to `npm run wdio-default`.
But, in order to disable the color contrast accessibility tests, I have added a separate wdio config for clinical-lowlight-theme which is being referenced from `wdio-lowlight` script.
Is this approach of adding a separate service for lowlight appropriate? "or"
Do we have any other alternatives?